invenio/modules/accounts/models.py

```diff
@@ -84,6 +84,16 @@ def __str__(self):
     PROFILE_FIELDS = ['nickname', 'email', 'family_name', 'given_names']
     """List of fields that can be updated with update_profile."""
 
+    @property
```

Comment:
I've just changed that :) What is the gain of having `hybrid_property`? (I read [here](http://docs.sqlalchemy.org/en/latest/orm/extensions/hybrid.html) that it can be used both at the class level and instance level, but here I guess we are only using it for instance level?)